frontend/packages/kubevirt-plugin/src/components/modals/cdrom-vm-modal/_cdrom-modal.scss

```diff
@@ -0,0 +1,32 @@
+.kv-cdrom-modal-padding {
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+.kv-cdrom-modal-restart-alert {
+  margin-bottom: 10px;
+}
+
+.kv-cdrom-modal-row {
+  .kv-cdrom-modal-row-edit {
+    display: grid;
+    grid-auto-flow: column;
+    grid-column-gap: var(--pf-global--gutter);
+
+    .kv-cdrom-modal-row-delete {
+      align-self: end;
+      justify-self: start;
+    }
+
+    .kv-cdrom-modal-row-value {
+      display: grid;
+      grid-auto-flow: column;
+      grid-column-gap: var(--pf-global--gutter);
+    }
+  }
```

Comment:
Could use just use the `pf-l-grid` system to do this rather than creating new classes for it? I've put this together here too: https://github.com/matthewcarleton/Openshift-Patternfly-demos/tree/master/layouts/Change-cd-modal